docs/modules/angular/pages/cookbooks/enable-standalone/enable-standalone-mode.adoc

```diff
@@ -0,0 +1,85 @@
+= Enable OneCX app to run in standalone
```

Comment:
Would be great if you could make sure to name the section's anchors - https://e-3d-dc1.capgemini.com/confluence/pages/viewpage.action?pageId=260114526#OneCXDocumentationGuidelines&Status-Structure&navigationguidelines